docs/parameters.yaml

```
@@ -2677,7 +2677,11 @@ components: ["origin", "cache", "director", "registry"]
 ############################
 name: Shoveler.Enable
 description: |+
-  Enable XRootD monitoring shoveler: https://github.com/opensciencegrid/xrootd-monitoring-shoveler.
+  The shoveler gathers UDP monitoring messages from XRootD servers and sends them to a reliable message bus, such
```

```suggestion
  The shoveler gathers UDP monitoring messages from XRootD servers and sends them to a message bus, such
```

Ideally it's reliable but that's really up to the federation operators / in the eye of the beholder. In general, I'd stay away from value judgements in technical docs unless they're concrete requirements.